shopsafe-frontend/src/app/components/feedback-forms/feedback-forms.component.css

```diff
@@ -0,0 +1,16 @@
+#title .heading-text {
+    padding-bottom: 20px;
+    text-align: center;
```

Comment:
could add `text-transform: uppercase` for consistency (my bad for not including this initially)

Reply:
It makes the text all uppercase